compiler/damlc/daml-ide/src/DA/Daml/LanguageServer/CodeLens.hs

```diff
@@ -52,16 +52,16 @@ handle ide (CodeLensParams (TextDocumentIdentifier uri) _) = Right <$> do
 
 -- | Convert a compiler virtual resource into a code lens.
 virtualResourceToCodeLens
-    :: (Range, T.Text, VirtualResource)
+    :: (Range, T.Text, T.Text, VirtualResource)
```

Comment:
We could consider using a sumtype for the kind and only render it in here but probably not worth doing.

Reply:
A bit of added type safety doesn't hurt.